.circleci/config.yml

```diff
@@ -460,6 +481,29 @@ jobs:
                   -timeout 60s \
                     python manage.py test
 
+  # Restore django and front POT files containing strings to translate and upload them to our
+  # translation management tool
+  upload-i18n-strings:
+    docker:
+      - image: circleci/node:10
+    working_directory: ~/marsha/scripts
+    steps:
+      - checkout:
+          path: ~/marsha
+      - run:
+          name: Install scripts dependencies
+          command: yarn install --frozen-lockfile
+      - attach_workspace:
+          at: ~/marsha/src/backend/locale
+      - run:
+          name: Use our node script to upload i18n backend strings file
+          command: node i18n/upload.js 'backend.pot' '/../../src/backend/locale/django.pot'
+      - attach_workspace:
+          at: ~/marsha/src/frontend/i18n
+      - run:
+          name: Use our node script to upload i18n frontend strings file
+          command: node i18n/upload.js 'frontend.pot' '/../../src/frontend/i18n/frontend.pot'
```

Comment:
Copy/pasta?

Reply:
Sorry it's not. But you can merge the two `attach_workspace` steps?

Reply:
No I can't, `at` property is not a list but a single `string`: https://circleci.com/docs/2.0/configuration-reference/#attach_workspace

Reply:
Okay, I missed the point. In this case, I would prefer attaching the workspace only once, but persist to the workspace a relevant path (see my other comment).

Reply:
Otherwise, that's an odd way to deal with the "workspace".